### ISSUE TITLE
Running ansible-playbook errors

### ISSUE THREAD

**Issue by julienchastang** [zonca/zonca.github.io-source#15]
```
fatal: [k8s-soars-2019-k8s-node-nf-4]: FAILED! => {"reason": "'delegate_to' is not a valid attribute for a TaskInclude\n\nThe error appears to be in '/home/openstack/jetstream_kubespray/roles/download/tasks/download_container.yml': line 2, column 3, but may\nbe elsewhere
in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n---\n- name: container_download | Make download decision if pull is required by tag or sha256\n  ^ here\n"}
```

https://github.com/zonca/zonca.github.io-source/blob/master/content/jetstream_kubernetes_kubespray.md#L138

**Comment by zonca** [comment]
have you updated ansible on your machine?

**Comment by julienchastang** [comment]
```
$ ansible --version
ansible 2.8.0
  config file = None
  configured module search path = [u'/home/openstack/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Apr  9 2019, 14:30:50) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
```

**Comment by zonca** [comment]
can you do a quick test switching to Ansible 2.7?

**Comment by julienchastang** [comment]
I had to replace this line

https://github.com/zonca/jetstream_kubespray/blob/branch_v2.8.2/requirements.txt#L1

with

```
ansible>=2.5.0,!=2.7.0,<2.8.0
```

**Comment by zonca** [comment]
ok, made the same change

**Comment by guanyuding** [comment]
I have just tested it. Both of these methods can be used. It is actually the same effect. and fixed my problem with ansible version 2.7.13. thx @zonca @julienchastang
